### PR TITLE
Render RSS Feed consistently between Writing settings screen and Podcasting Details screen

### PR DESCRIPTION
**Pull request by mattsherman**
- RSS Feed label was missing from Podcasting Details screen
- Learn more link was missing from Podcasting Details screen
- Font style and color of informational text under RSS feed differed between Writing settings screen and Podcasting Details screen

Before:

screencapture at thu jul 5 15 25 53 edt 2018

screencapture at thu jul 5 15 26 00 edt 2018

After:

screencapture at thu jul 5 15 23 30 edt 2018

screencapture at thu jul 5 15 23 40 edt 2018

To test:

With feature flag enabled (such as in development):

- View Writing settings screen and Podcasting Details screen and verify that RSS Feed and associated text is rendered the same on both screens

### CODE REVIEW

File: client/my-sites/site-settings/podcasting-details/style.scss [hunk 1]
```diff
@@ -172,3 +151,8 @@
 		margin: 12px 0 6px;
 	}
 }
+
+a.podcasting-details__support-link {
```
Comment (nylen) [inline client/my-sites/site-settings/podcasting-details/style.scss]: Let's not rely on this being an `a` tag.  The class name is specific enough to cover only this component, and the implementation of the `ExternalLink` component does sometimes return a `button` instead (IIRC) and its implementation may change further in the future.

Comment (mattsherman) [reply client/my-sites/site-settings/podcasting-details/style.scss]: The `a` in the selector was necessary to get the link underlined on the details page. I will create a separate issue to address this minor visual difference.

**Comment by mattsherman** [comment]
Thanks for the review and minor cleanup @nylen!

**Comment by dmsnell** [comment]
@mattsherman I stared at the before/after for a few minutes and I couldn't see any difference. Can you help a guy out and lead me to understand what changed?

**Comment by nylen** [comment]
@dmsnell:

- **Before, image 1 (Settings → Writing screen):** font color is gray, size is small.  "RSS Feed" heading is present.
- **Before, image 2 (Settings → Podcasting screen):** font color is black, size is larger.  "RSS Feed" heading is not present.
- **After**: font color is black, size is larger, "RSS Feed" heading is present.  Code for these two very similar sections is de-duplicated into a common component.  Also some further follow-up in #25913.

**Comment by mattsherman** [comment]
Thanks for replying @nylen ! 🙌

**Pull request merged**